hr_period/README.rst

```diff
@@ -0,0 +1,95 @@
+HR Payroll Period
```

Comment:
@pedrobaeza There is already a module named hr_payroll_period. That other module has like 15 dependencies and instead of only implementing payroll periods, it is a kind of wrapper around its dependencies.